CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Add `cljr-reify-to-defrecord`
 - Add `cljr-show-changelog` so users don't have to visit github to find out what's changed after a package update.
 - Add `cljr-create-fn-from-example` to create function stub based on example usage
+- now cljr--add-test-use-declarations actually checks the file system in order to find its require for the source ns.
```

Comment:
Capitalize this to match the other entries.  You could also do a slightly better job of explaining what's actually changed.  You've added support for various other test prefixes and suffixes instead of relying on the convention of `_test` suffix, right?